codespell_lib/tests/test_dictionary.py

```diff
@@ -189,6 +189,7 @@ def test_error_checking_in_aspell(err, rep, err_aspell, rep_aspell, match):
 # allow some duplicates, like "m-i-n-i-m-i-s-e", or "c-a-l-c-u-l-a-t-a-b-l-e"
 allowed_dups = {
     ('dictionary.txt', 'dictionary_en-GB_to_en-US.txt'),
+    ('dictionary.txt', 'dictionary_code.txt'),
```

Comment:
No longer needed, right?

Reply:
Probably not for the moment, although may appear again